tests/unit/auth/v2/queryAuthCheck.js

```diff
@@ -26,3 +27,94 @@ describe('v2: queryAuthCheck', () => {
         }
     }));
 });
+
+describe('v2: queryAuthCheck', () => {
+    let clock;
+
+    beforeEach(() => {
+        clock = sinon.useFakeTimers();
+    });
+    afterEach(() => {
+        process.env.PRE_SIGN_URL_EXPIRY = 604800000;
```

Comment:
Minor: for better readability, you might want to add a global variable; something like `const sevenHoursInMs = 7 * 24 * 60 * 60`;

Reply:
Good idea